StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostController.swift

```diff
@@ -83,13 +83,6 @@ extension HostController: HostViewControllerDelegate {
     ) {
         delegate?.hostController(self, didReceiveEvent: FinancialConnectionsEvent(name: .open))
 
-        guard
-            let consentPaneModel = synchronizePayload.text?.consentPane
-        else {
-            continueWithWebFlow(synchronizePayload.manifest)
-            return
-        }
-
```

Comment:
This code change may be a little confusing without a lot of context

Overall, it was unnecessary for it to live here because it assumes that we will always display the "consent screen before" (the screen with an "Agree" button to consent bank information sharing). This assumption is correct _most_ of the time, but there are some edge-cases where we may not want to display it as the first pane, so this change introduces flexibility.